README.md

```diff
@@ -72,7 +73,7 @@ No modules.
 
 The Terraform Azurerm Event Hub Namespace project is Copyright 2023-present Snowplow Analytics Ltd.
 
-Licensed under the [Apache License, Version 2.0][license] (the "License");
+Licensed under the [Snowplow Community License Agreement, Version 1.0][license] (the "License");
```

Comment:
I think lines 91 & 92 need to be updated as well to represent the correct link + badge still?